Channels/Operation/01_ManageOffers.md

```diff
@@ -0,0 +1,339 @@
+Note that any changes to an offer will trigger an automatic upload, so that any changes are applied to the products in the marketplace.
+In general, it is recommended to maintain or change offers that are mapped from a PIM product in the *PIM* module.
+
+#### Prerequisites
+
+At least one offer has been created, see [Create an offer](#create-an-offer).
+
+#### Procedure
+
+*Omni-Channel > Offers > Tab OFFERS*
+
+![Offers](../../Assets/Screenshots/Channels/Offers/Offers/Offers.png "[Offers]")
+
+1. Click the offer you want to edit in the list of offers.   
+    The *Edit offer* view is displayed. The *Attributes* tab is preselected.
+
+    > [Info] The tabs in the *Edit offer* view as well as the fields displayed in the *Attributes* tab differ depending on the selected connection.
+
+    ![Edit offer](../../Assets/Screenshots/Channels/Offers/Offers/EditOffer.png "[Edit offer]")
+
+2. Edit the desired data of the product in the corresponding fields and tabs.
+
+3. Click the [SAVE] button in the upper right corner.   
+    The changes have been saved. The *Edit offer* view is closed. The automatic upload has been triggered.
+
+    > [Info] Switch to *Offer Upload > Tab SCHEDULED UPLOADS* to check the queue for all scheduled offer uploads, see [Check the offer upload](./03_CheckOfferUpload.md).
+
+
+
+## Change the offer status
+
+The offer status specifies the behavior of the offer in the marketplace. By changing the offer status, you can provide an offer on a marketplace, hide an offer on a marketplace or remove an offer from a marketplace.
+It is recommended to change the offer status instead of deleting an offer to prevent any problems because of existing dependencies.   
+
+#### Prerequisites
+
+At least one offer has been created, see [Create an offer](#create-an-offer).
+
+#### Procedure
+
+*Omni-Channel > Offers > Tab OFFERS*
+
+![Offers](../../Assets/Screenshots/Channels/Offers/Offers/Offers.png "[Offers]")
+
+1. Click the checkbox of the offer whose status you want to change in the list of offers.   
+    The editing toolbar is displayed.
+
+2. Click the *Change status to* drop-down list in the editing toolbar and select the desired status. The following statuses are displayed:
+    - **Active**   
+        The offer is active. It is displayed in the marketplace, where it can be sold.   
+    - **Inactive**   
+        The offer is inactive. It is no longer displayed in the marketplace and thus cannot be sold anymore.   
+    - **Not available**   
+        The offer is deleted from the marketplace.      
+        The *Change offer status* confirmation window is displayed. If the variant status of the selected offer equals **Master offer**, an additional toggle is displayed within the confirmation window.   
```

Comment:
if its a child offer, that confirmatin window is also shown and asks if the selected status should also be applied to the parent product